src/users/dto/create-user.dto.ts

```diff
@@ -1,15 +1,18 @@
-import { IsNotEmpty, Matches, Max, Min } from 'class-validator';
+import {IsInt, IsNotEmpty, IsString, Matches, Max, Min} from 'class-validator';
 
 export class CreateUserDto {
   @IsNotEmpty()
+  @IsString()
   login: string;
 
   @Matches(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{2,}$/, {
     message: 'Password must contain letters and numbers',
   })
+  @IsString()
```

Comment:
I can use only letters and numbers, 
it is not possible to use other characters